demo/offline_bot/src/app/demo-adapter.ts

```diff
@@ -98,19 +99,25 @@ export class DemoAdapter extends ChatAdapter implements IChatGroupAdapter
                 toId: 999,
                 message: "Hi there, just type any message bellow to test this Angular module.",
                 dateSent: new Date()
-            }
+            },
+            {
+              fromId: 1,
+              toId: 999,
+              type: 3,
```

Comment:
We could probably have used the imported `MessageType ` instead of using the value `3` directly.

Reply:
Definitely you are right, I really didn't realize that one, didn't answer before cuz I was out of town. I hope soon I can add something else.